README.md

```diff
@@ -5,8 +5,8 @@
 
 # Introduction #
 
-*Uniqush* is a _free_ and _open source_ software system which provides a
-unified push service for server side notification to apps on mobile devices.
+*Uniqush* (pronounced: YOU-ni-kush) is a _free_ and _open source_ software system
```

Comment:
Might be interpreted as "knee" https://www.youtube.com/watch?v=zIV4poUZAQo

Alternately "uni" as in unified, "qush" as in cushion

http://www.merriam-webster.com/dictionary/unified
http://www.merriam-webster.com/dictionary/cushion